lib/src/components/NxDrawer/NxDrawer.tsx

```diff
@@ -12,18 +12,44 @@ import React, {
 } from 'react';
 import classnames from 'classnames';
 
+import NxOverflowTooltip from '../NxTooltip/NxOverflowTooltip';
 import AbstractDialog from '../AbstractDialog/AbstractDialog';
 import NxCloseButton from '../NxCloseButton/NxCloseButton';
 import withClass from '../../util/withClass';
 
-import { Props, NxDrawerHeaderProps, NxDrawerContextValue, OpenState, propTypes } from './types';
+import {
+  NxDrawerContextValue,
+  NxDrawerHeaderProps,
+  NxDrawerHeaderTitleProps,
+  OpenState,
+  Props,
+  propTypes
+} from './types';
 
 import './NxDrawer.scss';
 
 const NxDrawerContext = React.createContext<NxDrawerContextValue>({
   closeDrawer: () => {}
 });
 
+const NxDrawerHeaderTitle = (props: NxDrawerHeaderTitleProps) => {
```

Comment:
The prior implementation was able to accept a ref. This one should too to avoid a breaking change

Reply:
> The prior implementation was able to accept a ref. This one should too to avoid a breaking change

Thanks! I forwarded refs here: febd40de14d6f870840366f6a992f2ed1f0bb41b